samples/java-android-app/src/main/java/com/amplitude/android/sample/TourbleShootingPlugin.java

```diff
@@ -0,0 +1,28 @@
+package com.amplitude.android.sample;
+
+import com.amplitude.common.Logger;
+import com.amplitude.core.Amplitude;
+import com.amplitude.core.events.BaseEvent;
+import com.amplitude.core.platform.DestinationPlugin;
+import com.google.gson.Gson;
+
+public class TourbleShootingPlugin extends DestinationPlugin {
+    private Logger logger;
+    @Override
+    public void setup(Amplitude amplitude) {
+        logger = amplitude.getLogger();
+        String apiKey = amplitude.getConfiguration().getApiKey();
+        String serverZone = String.valueOf(amplitude.getConfiguration().getServerZone());
+        String serverUrl = amplitude.getConfiguration().getServerUrl();
+        logger.debug("Current Configuration : {\"apiKey\": "+apiKey+", \"serverZone\": "+serverZone+", \"serverUrl\": "+serverUrl+"}");
+        super.setup(amplitude);
+    }
+
+    @Override
+    public BaseEvent track(BaseEvent event) {
+        Gson gson = new Gson();
+        String eventJsonStr = gson.toJson(event);
+        logger.debug("Ingested Event: "+eventJsonStr);
```

Comment:
Ingested might be confusing here since the event may not be sent to the server. Maybe "Processed event" instead?